manifests/cert.pp

```diff
@@ -13,19 +14,26 @@
   # /etc/letsencrypt/live/${domain_name}/
   #
   #$options_hash = {
-  #  'cert_path' => "/etc/letsencrypt/live/${domain_name}/cert.crt",
-  #  'key_path' => "/etc/letsencrypt/live/${domain_name}/key.pki",
+  #  'cert_path'      => "/etc/letsencrypt/live/${domain_name}/cert.crt",
+  #  'key_path'       => "/etc/letsencrypt/live/${domain_name}/key.pki",
   #  'fullchain_path' => "/etc/letsencrypt/live/${domain_name}/fullchain.crt",
-  #  'chain_path' => "/etc/letsencrypt/live/${domain_name}/chain.crt",
+  #  'chain_path'     => "/etc/letsencrypt/live/${domain_name}/chain.crt",
   #}
   #$_formatted_hash = prefix($options_hash, '--')
   #$_options = join_keys_to_values($_formatted_hash, '=')
   #$flags = join($_options, ' ')
 
+  if($sub_domains) {
+    $prefixed_sub_domains = prefix($sub_domains, '-d ')
+    $letsencrypt_command = "letsencrypt certonly --register-unsafely-without-email --agree-tos --webroot --webroot-path ${webroot} -d ${domain_name} ${prefixed_sub_domains}"
+  } else {
+    $letsencrypt_command = "letsencrypt certonly --register-unsafely-without-email --agree-tos --webroot --webroot-path ${webroot} -d ${domain_name}"
```

Comment:
`--register-unsafely-without-email` seems like it would be bad for anyone who already has a letsencrypt cli.ini file already on the system. Instead of using this flag I have some code locally which will manage that configuration file so that when including the `letsencrypt_client` class you can pass in an email to recover the ssl cert in case something goes horribly wrong.